package.json

```diff
@@ -4,7 +4,7 @@
   "url": "http://github.com/kartotherian/node-mapnik",
   "homepage": "http://mapnik.org",
   "author": "Dane Springmeyer <dane@mapbox.com> (mapnik.org)",
-  "version": "3.7.3",
+  "version": "3.7.3-wikimedia.0",
```

Comment:
Please let us know when this is released to npm--I think that blocks the kartotherian patch.